CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# 1.4.20 (Nov 2020)
+  * XCode 12 support
+  * Completely reworked escape analysis for object allocation
+  * Use ForeignException wrapper to handle native exceptions ([KT-35056](https://youtrack.jetbrains.com/issue/KT-35056))
+  * CocoaPods plugin improvements
+  * equals/hashCode support for adapted callable references ([KT-39800](https://youtrack.jetbrains.com/issue/KT-39800))
+  * equals/hashCode support for fun interfaces ([KT-39798](https://youtrack.jetbrains.com/issue/KT-39798))
+  * IR-level optimizations
+    * Constant folding
+    * String concatenation flattenning
+    * Emit Lazy IR for cached structs and enums
+  * Various fixes to compiler caches such as
+    * Pre-link static caches ([KT-34261](https://youtrack.jetbrains.com/issue/KT-34261))
```

Comment:
It is in fact not ready and isn't enabled by default, so doesn't fix anything AFAIK.